client/src/components/DynamicJsonForm.tsx

```diff
@@ -36,15 +36,17 @@ interface DynamicJsonFormProps {
   value: JsonValue;
   onChange: (value: JsonValue) => void;
   maxDepth?: number;
+  onlyJSON?: boolean;
```

Comment:
When using the DynamicJSONForm as a fallback value, we won't have a full-featured schema and the Form view doesn't work. Added a new bool to disable the form view - open to feedback/ alternative approaches